Release 1.2.2 (#73)
nf-core/hic: README.md

nextflow run nf-core/hic -profile <docker/singularity/conda/institute> --reads '*_R{1,2}.fastq.gz' --genome GRCh37
```

> See [usage docs](docs/usage.md) for all of the available options when running

Reviewer: ```suggestion
See [usage docs](https://nf-co.re/hic/usage) for all of the available options when running
```

4. [Output and how to interpret the results](docs/output.md)
5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)

The nf-core/hic pipeline comes with documentation about the pipeline which

> you can read at [https://nf-core/hic/docs](https://nf-core/hic/docs) or

Reviewer 2: ```suggestion
you can read at [https://nf-core/hic/usage](https://nf-core/hic/usage or
```
See https://github.com/nf-core/tools/pull/728#issue-467862242

> you can read at [https://nf-core/hic/usage](https://nf-core/hic/usage) or

Reviewer: ```suggestion
you can read at [https://nf-co.re/hic/usage](https://nf-co.re/hic/usage) or
```
